buildenv/docker/mkdocker.sh

```diff
@@ -35,6 +35,8 @@ usage() {
   echo "  --cuda         include CUDA header files"
   echo "  --dist=...     specify the Linux distribution (e.g. centos, ubuntu)"
   echo "  --freemarker   include freemarker.jar"
+  echo "  --jdk=...      specify which jdk can be built (default: all)"
+  echo "  --nogitcache   don't generate the git cache"
```

Comment:
I think `--gitcache={yes|no}` would be more consistent.
If `yes`, the cache should be limited to openj9, omr and those versions specified in `--jdk=...`.